src/encoder.rs

```diff
@@ -1945,18 +1953,37 @@ fn encode_tile(fi: &FrameInvariants, fs: &mut FrameState) -> Vec<u8> {
             };
 
             assert!(!fi.sequence.use_128x128_superblock);
-            pmvs[1][r] = estimate_motion_ss2(
+            let mut pmvs1 = None;
+            let mut pmvs2 = None;
+            let mut pmvs3 = None;
+            let mut pmvs4 = None;
+            rayon::scope(|s| {
```

Comment:
Do you think the spawn() here helps for multi-threading?

Reply:
This is a special scope in which you spawn threads that will be joined at the end of the scope.

basically I'm spawning 4 threads and then wait fo them to finish.

Reply:
Indentation for the spawn sequence is uncomfortable although it minimizes the diff.
Please consider reformatting the resulting block.